sunpy/io/jp2.py

```diff
@@ -71,6 +72,101 @@ def get_header(filepath):
 
     return [FileHeader(pydict)]
 
+def header_to_xml(header):
```

Comment:
It might be easier to install pre-commit and run it locally. It has failed with a bunch of little style checks and doing it all automatically will be easier.

Reply:
Ok